### PR TITLE
Update Sidekiq to v6

### PR DESCRIPTION
**Pull request by jbirdjavi**
Testing here before upgrading other Rails apps.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -67,5 +67,5 @@ gem "lograge"
 gem "ougai", "~> 1.7"
 gem "rake"
 gem "redis-actionpack"
-gem "sidekiq", "< 6"
+gem "sidekiq"
```
Comment (mattdrees) [inline Gemfile:70]: probably supposed to be in the next commit. but that's fine.

**Review by mattdrees — APPROVED** [review]

File: Gemfile.lock [hunk 2]
```diff
@@ -210,14 +208,13 @@ GEM
...
+    sidekiq (6.1.2)
+      connection_pool (>= 2.2.2)
       rack (~> 2.0)
-      rack-protection (>= 1.5.0)
-      redis (>= 3.3.5, < 4.2)
-    sidekiq-pro (5.0.0)
-      concurrent-ruby (>= 1.0.5)
-      sidekiq (>= 5.2.7)
+      redis (>= 4.2.0)
+    sidekiq-pro (5.2.0)
```
Comment (mattdrees) [inline Gemfile.lock:215]: sidekiq-pro doesn't need to match, right?

Comment (jbirdjavi) [reply Gemfile.lock:215]: Correct, they have different version numbers.

File: Gemfile (hunk 1 above)
Comment (jbirdjavi) [reply Gemfile:70]: Yup, it was supposed to be.

**Pull request merged**